packages/shared/src/storage/GoogleDriveStorage/GoogleDriveStorage.android.ts

```
@@ -0,0 +1,265 @@
+  async uploadFile(params: {
+    fileName: string;
+    content: string;
+  }): Promise<{ fileId: string }> {
+    const { fileName, content } = params;
+
+    // Ensure user is signed in
+    const signedIn = await GoogleSignin.isSignedIn();
+    if (!signedIn) {
+      throw new OneKeyLocalError(
+        'Not signed in to Google. Please sign in first.',
+      );
```

ESLint

[error] Expected an error object to be thrown.

(@typescript-eslint/no-throw-literal)